tests/virt_autotest/sriov_pci_passthrough.pm

```diff
@@ -0,0 +1,352 @@
+# SUSE's openQA tests
```

Comment:
I have to say if this module is only meant to work with network card. The module name is too general.Please use sriov_pci_passthrough_network or similar instead

Reply:
Yes, good suggestion. I just messed up the push I will change the name after I clean up the commits.